### PR TITLE
Contrib whoosh

### PR DESCRIPTION
**Pull request by tloizou**

### CODE REVIEW

**Comment by boegel** [comment]
Jenkins: ok to test

File: easybuild/easyconfigs/a/astropy/astropy-1.0.6-goolf-1.4.10-Python-2.7.3.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+
+python = 'Python'
+pyver = '2.7.3'
+versionsuffix = '-%s-%s' % (python, pyver)
+
+dependencies = [
+    (python, pyver),
+    ('setuptools', '18.4', versionsuffix),
```
Comment (boegel) [inline easybuild/easyconfigs/a/astropy/astropy-1.0.6-goolf-1.4.10-Python-2.7.3.eb]: @tloizou easyconfig for this is missing, any idea if it's available somewhere else, e.g. a PR that isn't merged yet?

Comment (tloizou) [inline easybuild/easyconfigs/a/astropy/astropy-1.0.6-goolf-1.4.10-Python-2.7.3.eb]: @boegel: sorry this files shouldn't be here. I will correct this. Please see PR #2724 for astropy which includes the setuptools config.

Comment (boegel) [inline easybuild/easyconfigs/a/astropy/astropy-1.0.6-goolf-1.4.10-Python-2.7.3.eb]: OK, thanks!

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2159.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/404ec146ad0ada301fb7 for a full test report.

**Comment by boegel** [comment]
lgtm, going in, thanks @tloizou!

**Pull request merged**